starter_code/js/app.js

```diff
@@ -0,0 +1,25 @@
+$( document ).ready(function(e) {
+
+	$('.readmore').click(function(e){
+		$('.readmore').hide();
+		$('#show-this-on-click').slideDown();
+		$('.readless').removeClass('hide');
+		e.preventDefault();
```

Comment:
It's better to put the ```preventDefault()``` at the top of the function rather than at the bottom to show that before we do anything, we prevent the default behaviour.